external-crates/move/move-compiler/src/command_line/compiler.rs

```diff
@@ -743,6 +743,36 @@ fn has_compiled_module_magic_number(path: &str) -> bool {
     num_bytes_read == BinaryConstants::MOVE_MAGIC_SIZE && magic == BinaryConstants::MOVE_MAGIC
 }
 
+pub fn move_check_for_errors(
```

Comment:
Not really relevant for this PR just a comment:
We might want to rename it and flesh out this API (as it also does warnings now... this function is super old fwiw)

Reply:
Good